setup.cfg

```diff
@@ -95,6 +95,13 @@ docs =
 sbml =
     python-libsbml
     lxml
+sbmlmath =
+    # sbmlmath is py3.9+ only. For version below 3.9, use the following install command locally:
+    # pip install sbmlmath@git+https://github.com/dweindl/sbmlmath.git --python-version py39 --ignore-requires-python
+    # It might be necessary to install sbmlmath, pint and lxml using --no-deps as well.
+    pint
+    lxml>=4.6.4
+    sbmlmath @ git+https://github.com/dweindl/sbmlmath.git
```

Comment:
Install from PyPI

Reply:
Resolved in https://github.com/indralab/mira/pull/188/commits/37ee300b37a3d477014a3d9d7d751c2f17d4b0bb